styleguide/derek/index.jade

```diff
@@ -19,16 +19,12 @@
 .row.half-padding-top
   .col-md-12
     h6 helpful links
-    ul.nav.nav-pills.nav-stacked
+    ul
       li
-        a(href="derek/solutions") Solutions
+        a.red(href="derek/templates") Templates
       li
-        a(href="derek/templates") Templates
+        a.red(href="/derek/patterns/patterns") Patterns and how to use them
       li
-        a(href="derek/incubation") Dev Sandbox/Incubation
-
-.row.half-padding-top
-  .col-md-12
-    h2.red Our Story
-  .col-md-8
-    p The story how zoolander came to be. Its mission and the global ecomm manifesto
+        a.red(href="derek/globalTheme/icon-font") Icon Font
+      li
+        a.red(href="https://raxglobal.sharepoint.com/Global%20Ecommerce/Shared%20Documents/Forms/AllItems.aspx?id=%2FGlobal%20Ecommerce%2FShared%20Documents%2FeComm%20Ops%2FWeb%20page%20changes%2FTemplate%20Documentation") Writing Guide
```

Comment:
Let's mask this URL. You can use https://rax.io/create to create shortened/obfuscated URLs that only resolve in-network.